apps/jwt_auth/tests/test_apis.py

```diff
@@ -37,3 +38,15 @@ def test_refresh_token(self):
 
         response = self.client.post(url, data)
         self.assert_200(response)
+
+    def test_login_with_password_greater_than_128_characters(self):
+        data = {
+            'username': "Hari@gmail.com",
+            "password": 'a'*130
+        }
+        url = '/api/v1/token/'
+
+        response = self.client.post(url, data)
+        self.assert_400(response)
+        assert 'password' in response.data['errors']
+        self.assertEqual(response.data['errors']['password'][0], 'This password has exceed the limit of 128 characters')
```

Comment:
Tallying messages is okay and not okay. I will go with @thenav56.

Reply:
Yeah, Line 51 is sufficient.